### PR TITLE
Problem: Generated Makefile had strange rules for src: and code:

### PR DESCRIPTION
**Pull request by jimklimov**
Solution: My guess at how to fix it, real review is desirable :)

@sappo : you seem to have latest git-blame hits in this area, plz take a look before just merging :)

### CODE REVIEW

**Comment by jimklimov** [comment]
Diff of generated codes : 

````
 # define custom target for all products of /src
-src:
-       src/fty-metric-store \
-       src/fty_metric_store_selftest \
-       src/libfty_metric_store.la
-
-       cd $(srcdir); gsl -target:- project.xml
-
+src: \
+               src/fty-metric-store \
+               src/fty_metric_store_selftest \
+               src/libfty_metric_store.la
````

So now `src` depends on those listed binaries and libs (rather than trying to run them - which was probably not intended), and there is no dangling `gsl` call outside any recipe...

File: zproject_autotools.gsl [hunk 1]
```diff
@@ -1323,11 +1323,11 @@ dist_api_DATA = \\
 .endfor
 .if count (project.class)
 # define custom target for all products of /src
-src:
+src: \\
```
Comment (sappo) [inline zproject_autotools.gsl]: This looks wrong. Correct me if I'm wrong but make targets don't continue on next line.

File: zproject_autotools.gsl [hunk 2]
```diff
@@ -1323,11 +1323,11 @@ dist_api_DATA = \\
...
 .   for project.main
-\tsrc/$(main.name) \\
+\t\tsrc/$(main.name) \\
 .   endfor
-\tsrc/$(project.libname).la
+\t\tsrc/$(project.libname).la
```
Comment (sappo) [inline zproject_autotools.gsl]: Why add a second tab. This will kill the formatting.

File: zproject_autotools.gsl (hunk 1 above)
Comment (jimklimov) [inline zproject_autotools.gsl]: You can concatenate long lines, at least in many `make`s... Can probably make it generate a single-line in GSL though.

The code generated earlier defined a target called `src` with no dependencies and whose action was to run a single command made up of names of binaries and the `.la` lib. Now, THAT certainly seemed wrong :)

File: zproject_autotools.gsl (hunk 2 above)
Comment (jimklimov) [inline zproject_autotools.gsl]: I thought on the opposite - at least initially, where it was hard to discern the (presumed) dependencies from the recipe rules (the `gsl` call below).

Anyhow, if we concatenate the `src` deps into one line, this nit will disappear too :)

File: zproject_autotools.gsl (hunk 1 above)
Comment (sappo) [inline zproject_autotools.gsl]: Yes you're right, just checked this command in zyre. It doesn't do anything.

**Pull request merged**